src/components/new-members/index.js

```diff
@@ -6,23 +6,42 @@ import { membersContext } from '@store/members/members-context';
 import { searchMemberContext } from '@store/search-members/searchMembers-context';
 import { searchMembers } from '@helper-functions/search-members';
 
+const cardColorArray = [
+  { color_primary: '#DB1212', color_secondary: '#F88181' },
+  { color_primary: '#E9C46A', color_secondary: '#FFE094' },
+  { color_primary: '#F4A261', color_secondary: '#FFBF8C' },
+  { color_primary: '#2A9D8F', color_secondary: '#42E0CD' },
+  { color_primary: '#165692', color_secondary: '#3D8BD3' },
+  { color_primary: '#264653', color_secondary: '#387892' },
+];
+let counter = 0;
+
 // returns card which shows details of new member
-const renderNewMember = (newMember, isOptionKey) => (
-  <div className={styles.containerForNewMember}>
-    <Card
-      developerInfo={newMember}
-      isMember={false}
-      isOptionKey={isOptionKey}
-    />
-  </div>
-);
+const renderNewMember = (newMember, isOptionKey) => {
+  const prev = counter;
+  counter += 1;
+  if (counter > cardColorArray.length - 1) counter = 0;
+  return (
+    <div className={styles.containerForNewMember}>
+      {process.browser && (
+        <Card
+          developerInfo={newMember}
+          isMember={false}
+          isOptionKey={isOptionKey}
+          colorCombination={cardColorArray[prev]}
```

Comment:
instead of prev which is a non-reactive state, use a random function that generates the random no. between a range and pass that
```Math.floor(Math.random() * (cardColorArray.length))```

Reply:
I would have done that In the first place, but since we decided on some kind of a pattern to be used I passed the colors in a sequence. But i dont get why `non-reactive state is important here`. We are not re-rendering the cards again once they are loaded.

Reply:
It's a good practice in my opinion to be safe from the edge cases

Reply:
can you elaborate more on this , help me get a better context

Reply:
I also want to remind @vickydonor-99, we want some sort of hashing function, that distributes the colors, but makes it such that the same color is always applied to the same username.

Reply:
yes @ankushdharkar, haven't got a clear idea about it yet